configure.ac

```
@@ -2911,6 +2941,81 @@ AS_CASE([$target],
     # as "Infinity" and "Inf" instead of the expected "inf"
     [AC_DEFINE([HAS_BROKEN_PRINTF], [1])])
 
+# Determine the three Runtime IDs (see runtime/Mangling.md)
+m4_define([ALPHABET], [0123456789abcdefghijklmnopqrstuv])
+alphabet='ALPHABET'
```

How about
```suggestion
AC_DEFUN([BASE32],
  ["$(echo 'ALPHABET' | cut -c $(expr $1 + 1))"])
```
so that later we can simply say:
```
quintet1=BASE32([$quintet1])
```

---

Much nicer, yes! Bordering on readable, but let's not get ahead of ourselves - it's still m4sh!